client/appengine_devices.go

```diff
@@ -152,3 +152,42 @@ func (s *AppEngineService) GetDevicesStats(realm string) (DevicesStats, error) {
 
 	return deviceStats, err
 }
+
+// ListDeviceMetadata is an helper to list all Metadata of a Device
+func (s *AppEngineService) ListDeviceMetadata(realm string, deviceIdentifier string, deviceIdentifierType DeviceIdentifierType) (map[string]string, error) {
```

Comment:
If many args share the same type, instead of writing
```
somefun(arg1 type, arg2 type)
```
a more compact form is supported:
```
somefun(arg1, arg2 type)
```
The same comment applies to the other function definitions